### PR TITLE
Bugfix with chain lookup

### PR DESCRIPTION
**Pull request by Ptroger**
This PR stop gathering 'AssetIds' for price feed.
The subsequent call to `getPrices` method is hardcoded to return an empty object. So the solution here is to also force the return of an empty array in `getAssetIds` method instead of trying to build a list of assetsIds.

This is the only place where `chains` constant seems to be used. So maybe we can deprecate it at the scope of this PR, and find another solution to gather prices later

### CODE REVIEW

File: apps/armory/src/data-feed/core/service/__test__/unit/price-feed.service.spec.ts [hunk 1]
```diff
@@ -80,13 +80,14 @@ describe(PriceFeedService.name, () => {
       })
     })
 
-    it('calls price service', async () => {
-      await service.getFeed(authzRequest)
+    // 15-08-2024 @mattschoch - skipping because price feed is currently disabled
```
Comment (wcalderipe) [inline apps/armory/src/data-feed/core/service/__test__/unit/price-feed.service.spec.ts:83]: Skip the test, don't delete it.

File: packages/transaction-request-intent/src/lib/utils.ts [hunk 2]
```diff
@@ -330,7 +330,32 @@ export const checkCancelTransaction = (input: NativeTransferInput): Intents => {
...
+      coinType = Slip44SupportedAddresses.FTM
+      break
+    case SupportedChains.POLYGON:
+      coinType = Slip44SupportedAddresses.MATIC
+      break
+    case SupportedChains.OPTIMISM:
+      coinType = Slip44SupportedAddresses.ETH
+      break
+    default:
+      coinType = Slip44SupportedAddresses.ETH
```
Comment (wcalderipe) [inline packages/transaction-request-intent/src/lib/utils.ts:357]: You can replace this imperative massive switch block by a record. Moreover, you don't need a variable `coinType` if you just return the value – in general, don't create state if you don't need it.

```typescript
const coins: Record<number, AssetId> = {
  [SupportedChains.CELO]: Slip44SupportedAddresses.CELO,
  [SupportedChains.BNB]: Slip44SupportedAddresses.BNB,
  // ...
}

return coins[chainId] ? coins[chainId] :  Slip44SupportedAddresses.ETH
```

File: packages/transaction-request-intent/src/lib/domain.ts [hunk 3]
```diff
@@ -155,12 +155,22 @@ export type AssetTypeAndUnknown = AssetType | Misc.UNKNOWN
...
 export enum Slip44SupportedAddresses {
   ETH = 60,
-  MATIC = 966
+  MATIC = 966,
+  BNB = 714,
+  FTM = 1007,
+  ARBITRUM = 9001,
+  AVALANCHE = 9000,
+  CELO = 52752,
 }
```
Comment (wcalderipe) [inline packages/transaction-request-intent/src/lib/domain.ts:174]: What if we inject them from the caller as a [decode config](https://github.com/narval-xyz/armory/blob/efe229a548afca3d7df35c659fd21fee64b21797/packages/transaction-request-intent/src/lib/decoders/decode.ts#L141-L141)?

File: apps/armory/src/data-feed/core/service/__test__/unit/price-feed.service.spec.ts (hunk 1 above)
Comment (mattschoch) [reply apps/armory/src/data-feed/core/service/__test__/unit/price-feed.service.spec.ts:83]: We have a lint rule preventing `.skip` on tests lol

**Review by mattschoch — APPROVED** [review]
Going to approve/merge this as a bugfix, but we'll keep going on this asap to get it cleaned up.
But tested this as-is and fixed the OP bug(s)

**Pull request merged**